jsapp/js/components/permissions/transferProjects/inviteBanner.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Icon from 'js/components/common/icon';
+import Button from 'js/components/common/button';
+import {TransferStatuses} from 'js/components/permissions/transferProjects/transferProjects.api';
+import styles from './inviteBanner.module.scss';
+
+export interface InviteState {
+  valid: boolean;
+  uid: string;
+  status: TransferStatuses.Accepted | TransferStatuses.Declined | null;
+  name: string;
+  currentOwner: string;
+}
+
+interface InviteBannerProps {
+  invite: InviteState;
+  onRequestClose: () => void;
+}
+
+/**
+ * Displays a banner about accepting or declining project transfer invitation.
+ */
+export default function InviteBanner(props: InviteBannerProps) {
```

Comment:
For the organizations project we are going to have organization membership invites that will be very similar to project ownership transfer invites. I haven't thought about it too much, but I don't think we will reuse this component for that. With that in mind, do you think we should give the components in the transfer projects folder more specific names so we don't confuse them with other invite flows down the line? So in this case maybe something like ProjectTransferInviteBanner. It's a bit wordy, but might be good to avoid confusion.